tutoraspects/patches/kustomization-configmapgenerator

```diff
@@ -68,6 +68,13 @@
     labels:
         app.kubernetes.io/name: superset
 
+- name: superset-localization
+  files:{% for file in "aspects/apps/superset/localization"|walk_templates %}
+  - plugins/{{ file }}{% endfor %}
+  options:
+    labels:
+        app.kubernetes.io/name: superset
```

Comment:
nit: update label `superset-localization`

Reply:
The others around it are labeled this way, should they be changed as well?

Reply:
Yes, they should